spec/models/user_profile_spec.rb

```diff
@@ -1,6 +1,43 @@
 # frozen_string_literal: true
 
-describe UserProfile do
+RSpec.describe UserProfile do
```

Comment:
What's the pros of prefixing this line with `RSpec`?

Reply:
It’s just the non-deprecated way of doing things, it has been for years now :sweat_smile: I think this was introduced when they provided a way of not having to patch the whole world to use rspec (`expect` instead of `should`).